msal/tests/Test.MSAL.NET.Integration/DeviceCodeIntegrationTests.cs

```diff
@@ -45,6 +47,7 @@ public class DeviceCodeIntegrationTests
 
         [TestMethod]
         [TestCategory("UsernamePasswordIntegrationTests")]
+        [Ignore]
         public async Task AcquireTokenWithManagedUsernamePasswordAsync()
```

Comment:
>AcquireTokenWithManagedUsernamePasswordAsync [](start = 26, length = 44)

@jennyf19: Not sure why this test is in a test class for DeviceCodeIntegrationTests, thoughts?

Reply:
Not sure. The actual u/p integration tests are in a [separate class](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/blob/dev/msal/tests/Test.MSAL.NET.Integration/UsernamePasswordIntegrationTests.cs#L39) - I think the method name and TestCategory name did not get changed on a copy/paste. just a guess. want me to fix it? 

---
In reply to: [223430145](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1304#discussion_r223430145) [](ancestors = 223430145)

Reply:
)

Reply:
copy paste == evil